Microsoft.Toolkit.Uwp.UI/Cache/CacheBase.cs

```diff
@@ -212,7 +213,7 @@ public Task PreCacheAsync(Uri uri, bool throwOnError = false, bool storeToMemory
         /// <returns>an instance of Generic type</returns>
         public Task<T> GetFromCacheAsync(Uri uri, bool throwOnError = false)
         {
-            return GetItemAsync(uri, throwOnError, false);
+            return GetItemAsync(uri, throwOnError, false, default(CancellationToken));
```

Comment:
That was the only one requested.. You are right should make both methods cancellable